Boardfiles/nucleol552zeq/Core/Src/usart.c

```diff
@@ -174,16 +174,18 @@ void MX_USART2_UART_Init(void)
 
   /* USER CODE END USART2_Init 1 */
   huart2.Instance = USART2;
-  huart2.Init.BaudRate = 115200;
-  huart2.Init.WordLength = UART_WORDLENGTH_8B;
-  huart2.Init.StopBits = UART_STOPBITS_1;
-  huart2.Init.Parity = UART_PARITY_NONE;
+  huart2.Init.BaudRate = 100000;
+  huart2.Init.WordLength = UART_WORDLENGTH_9B;
+  huart2.Init.StopBits = UART_STOPBITS_2;
+  huart2.Init.Parity = UART_PARITY_EVEN;
   huart2.Init.Mode = UART_MODE_TX_RX;

```

Comment:
In your PR description, could you please explain what board settings you have changed, and why you have changed them? Looking at this, I dont understand why we would use a non-standard  BaudRate of 100000

Reply:
These settings are from the SBUS documentation outlining the desired configurations. Heres a link to that https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/2251456686/SBUS+Receiver